SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -701,6 +702,7 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 			OutgoingOnCreateEntityRPCs.FindOrAdd(TargetObject).RPCs.Add(Payload);
 #if !UE_BUILD_SHIPPING
+			NETWORK_PROFILER(GNetworkProfiler.TrackSendRPC(Channel->Actor, Function, 0, Payload.CountDataBits(), 0, NetDriver->GetSpatialOSNetConnection()));
```

Comment:
Thoughts on embedding this inside SpatialMetrics?

Reply:
@dbsigurd ^^

Reply:
Sorry, I don't know what happened to my replies. I could see them, but it seems like they didn't send and were just saved or something.

Anyways, I put these here as it most closely resembles how Epic set it up on their net driver. Having them in the same place you'd expect as Epic should help porting the 4.24 integration, and should always be there as long as Epic supports the tool, where the spatial metrics feels like a more ad hoc. Also the spatial metric tool doesn't support property updates, calculating waste, or time replicating.

Reply:
I was more referring to embedding the TrackSendRPC call inside spatial metrics, as now we are tracking every RPC in two places. If we're going to keep it inside SpatialSender, please pull it into a utility function, so we can do all in one spot.